applovin_max/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## x.x.x
+* Add platform implementation for adaptive banner.
```

Comment:
I think this should be more reflective of the change from a product perspective, as opposed to technically what you did.